src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -1106,7 +1106,11 @@ ExitInfo ExecutorWorker::checkLiteSyncInfoForEdit(SyncOpPtr syncOp, const SyncPa
                             syncOp->affectedNode()->lastmodified().has_value() ? *syncOp->affectedNode()->lastmodified() : 0,
                             syncOp->affectedNode()->size(),
                             syncOp->affectedNode()->id().has_value() ? *syncOp->affectedNode()->id() : std::string(), error);
+                    // TODO: Vfs functions should return an ExitInfo struct
                     syncOp->setOmit(true); // Do not propagate change in file system, only in DB
+                    if (!error.empty()) {
+                        return {ExitCode::SystemError, ExitCause::FileAccessError};
```

Comment:
In `VfsMac` we can have error such as `Bad parameters`, ... Is it really relevant to return `ExitCause::FileAccessError` anyway every time?

Reply:
No, that's why there is a TODO just before and a new Jira issue:
https://infomaniak.atlassian.net/browse/KDESKTOP-1359?atlOrigin=eyJpIjoiNjIwYjRmMDUxNjIyNDE2Njk2ZTE4YjNiZDg5N2Q3YzgiLCJwIjoiaiJ9
But in 99% of cases this fix should work...

Reply:
Okay 👍